### DISCUSSION TITLE
Using PYSTAL via a Dataframe

### DISCUSSION THREAD

**Discussion by salsa360** [pycaret/pycaret#1455] [Q&A]

**Comment by salsa360** [comment]
Hi Nikhil,

How do I modify the train test split?  I tried using the sktime line below but it does not work:

y_train, y_test = temporal_train_test_split(y, test_size=0.65)

It still opts for the default 'y_train' setting that you have.

Thanks.

Regards
Kushwant

**Comment by ngupta23** [comment]
Hi Kushwant,

I hope you are referring to the time series alpha module release and not pystal specifically.

Currently, we take care of train test split internally based on the forecast horizon that you provide. This is done since this is time series and you can not randomly split the test set. You need to maintain the temporal dependence in the data. Hope that helps.
